### PR TITLE
[PUB-3248] Allow Share Again for all Customers

### PR DESCRIPTION
**Pull request by hamstu**
## Description

https://buffer.atlassian.net/browse/PUB-3248

## Checklist

-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/server/parsers/orgParser.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ module.exports = orgData => ({
   hasBitlyFeature: orgData.planBase !== 'free',
   has30DaySentPostsLimitFeature: orgData.planBase === 'free', // profiles_controller updates_sent() returns only 30 days of sent posts for free users.
   hasCalendarFeature: orgData.planBase !== 'free',
-  hasShareAgainFeature: orgData.planBase !== 'free',
+  hasShareAgainFeature: true, // Feb 2021 - All plans can use Share Again after Pinterest depaywall: https://buffer.atlassian.net/browse/PUB-3248
```
Comment (albennett) [inline packages/server/parsers/orgParser.js:33]: ♻️ If everyone has the shareAgain feature, let's fully remove the check. 🗑️

Comment (anafilipadealmeida) [reply packages/server/parsers/orgParser.js:33]: I agree with @albennett here 🙂 Assuming this is not an experiment, it's worth removing the `hasShareAgainFeature` from the 7 files!

Comment (hamstu) [reply packages/server/parsers/orgParser.js:33]: Thanks @anafilipadealmeida and @albennett! Sounds like a plan. I'll update that today. 🙏🏻

File: packages/composer/interfaces/buffer-publish.jsx [hunk 2]
```diff
@@ -47,9 +47,7 @@ const ComposerWrapper = ({
 
   const saveButtons = getSaveButtons();
 
-  // Add Share Next feature to all users except free.
-  const showShareNextFeature = organizations.selected?.hasShareNextFeature;
-  if (showShareNextFeature && !editMode && !draftMode && !emptySlotMode) {
+  if (!editMode && !draftMode && !emptySlotMode) {
```
Comment (albennett) [inline packages/composer/interfaces/buffer-publish.jsx:50]: 💭 `shareNext` and `shareAgain` are different, right? 🤔

Comment (hamstu) [reply packages/composer/interfaces/buffer-publish.jsx:50]: Whoops, my search in VS Code was a bit too agressive. 🙈 

Let me pull that one back.

**Review by albennett — APPROVED** [review]
Thanks for removing all the references 🚀 LGTM once you add back `shareNext`

**Pull request merged**